iron-list.html

```diff
@@ -919,7 +919,7 @@
         // First element child is item; Safari doesn't support children[0]
         // on a doc fragment.
         physicalItems[i] = inst.root.querySelector('*');
-        Polymer.dom(this).appendChild(inst.root);
+        Polymer.dom(this._userTemplate.parentNode).appendChild(inst.root);
```

Comment:
`this._itemsParent.appendChild`. Also in L1759: `this._itemsParent.appendChild(stampedTemplate.root);`